src/common/csirf.cpp

```diff
@@ -25,6 +25,7 @@ limitations under the License.
 #include "sirf/iUtilities/DataHandle.h"
 #include "sirf/common/DataContainer.h"
 #include "sirf/common/ImageData.h"
+#include "../../Synergistic/cSyn/include/sirf/Syn/utilities.h"
```

Comment:
This shouldn't be done as it is.

Reply:
Had to do it this way because the build failed on Windows when I used the correct one. Magically, now builds ok! Could be something wrong with some CMakeLists actually.

Reply:
If this is a new folder (I think `Syn` didn't exist before), then you need to do `target_link_directories` in CMake.

Reply:
Apologies, I see that you've actually done that.

Reply:
Actually, I was wrong about 'now builds ok' - ok on Windows and VM, but some Travis builds fail with
```
fatal error: sirf/Syn/utilities.h: No such file or directory
```